### PR TITLE
L2D17A2

### PR DESCRIPTION
**Pull request by evyryan**
Added question headers
fixed fatheads
one line break error

### CODE REVIEW

File: kzz7yh-f51825/cod-ve09v2_kzz7yh-f51825.xml [hunk 1]
```diff
@@ -85,13 +85,14 @@
...
+               <head type="question-title">Utrum aliquid de natura celesti fiat de essentia ipsius ade</head>
                <p xml:id="kzz7yh-f51825-d1e150">
       Questio. I 
       <lb ed="#V" n="33"/>PRimo ostendo quod aliquid de 
       na<lb ed="#V" n="34" break="no"/>tura celesti fiat 
-      <lb ed="#V" n="35"/>e essentia ipsius ade philosophus. 16 de animalibus. 
-      di<lb ed="#V" n="36" break="no"/>cit quod in semine triplex est calor scilicet celestis igneus et 
-      <lb ed="#V" n="37"/>animalis. Sed accidens non est stne subiecto: ergo 
+      <lb ed="#V" n="35"/>de essentia ipsius ade <ref>philosophus. 16 de animalibus.</ref> 
+      di<lb ed="#V" n="36" break="no"/>cit quod in semine<!--s or f? see line below too--> triplex est calor scilicet celestis igneus et
```
Comment (RobbieBuhite) [inline kzz7yh-f51825/cod-ve09v2_kzz7yh-f51825.xml:94]:
```suggestion
      di<lb ed="#V" n="36" break="no"/>cit quod in semine triplex est calor scilicet celestis igneus et 
```

File: kzz7yh-f51825/cod-ve09v2_kzz7yh-f51825.xml [hunk 2]
```diff
@@ -85,13 +85,14 @@
...
                <p xml:id="kzz7yh-f51825-d1e150">
       Questio. I 
       <lb ed="#V" n="33"/>PRimo ostendo quod aliquid de 
       na<lb ed="#V" n="34" break="no"/>tura celesti fiat 
-      <lb ed="#V" n="35"/>e essentia ipsius ade philosophus. 16 de animalibus. 
-      di<lb ed="#V" n="36" break="no"/>cit quod in semine triplex est calor scilicet celestis igneus et 
-      <lb ed="#V" n="37"/>animalis. Sed accidens non est stne subiecto: ergo 
+      <lb ed="#V" n="35"/>de essentia ipsius ade <ref>philosophus. 16 de animalibus.</ref> 
+      di<lb ed="#V" n="36" break="no"/>cit quod in semine<!--s or f? see line below too--> triplex est calor scilicet celestis igneus et 
+      <lb ed="#V" n="37"/>animalis. Sed accidens non est stne<!--word--> subiecto: ergo
```
Comment (RobbieBuhite) [inline kzz7yh-f51825/cod-ve09v2_kzz7yh-f51825.xml:95]: The word should be "sine" but the type print is a little wonky, so I am unsure of whether I should make the definitive correction.

File: kzz7yh-f51825/cod-ve09v2_kzz7yh-f51825.xml [hunk 3]
```diff
@@ -304,8 +308,8 @@
       pun<lb ed="#V" n="44" break="no"/>ctualem equalitatem. 
    </p>
                <p xml:id="kzz7yh-f51825-d1e551">
-                  <lb ed="#V" n="45"/>Contra nios videmus quod secndum rectum omplexionem 
-      cor<lb ed="#V" n="46" break="no"/>debet esse magis calidum quam frigidum: et cerebrum 
+                  <lb ed="#V" n="45"/>Contra nos videmus quod secndum rectum complexionem
```
Comment (RobbieBuhite) [inline kzz7yh-f51825/cod-ve09v2_kzz7yh-f51825.xml:311]:
```suggestion
                  <lb ed="#V" n="45"/>Contra nos videmus quod secundum rectum complexionem 
```

File: kzz7yh-f51825/cod-ve09v2_kzz7yh-f51825.xml [hunk 4]
```diff
@@ -304,8 +308,8 @@
       pun<lb ed="#V" n="44" break="no"/>ctualem equalitatem. 
    </p>
                <p xml:id="kzz7yh-f51825-d1e551">
-                  <lb ed="#V" n="45"/>Contra nios videmus quod secndum rectum omplexionem 
-      cor<lb ed="#V" n="46" break="no"/>debet esse magis calidum quam frigidum: et cerebrum 
+                  <lb ed="#V" n="45"/>Contra nos videmus quod secndum rectum complexionem 
+      cor<!--word?--><lb ed="#V" n="46" break="no"/>debet esse magis calidum quam frigidum: et cerebrum
```
Comment (RobbieBuhite) [inline kzz7yh-f51825/cod-ve09v2_kzz7yh-f51825.xml:312]:
```suggestion
      cor<lb ed="#V" n="46" break="no"/>dam esse magis calidum quam frigidum: et cerebrum 
```

File: kzz7yh-f51825/cod-ve09v2_kzz7yh-f51825.xml [hunk 5]
```diff
@@ -421,12 +425,13 @@
             </div>
             <div xml:id="kzz7yh-f51825-Dd1e778">
                <head xml:id="kzz7yh-f51825-Hd1e780">Quaestio 4</head>
+               <head type="question-title">Utrum partes ipsius ade ante separationem anime a corpe fuerunt plures res in actu</head>
                <p xml:id="kzz7yh-f51825-d1e782">
       Quaesio IIII.
```
Comment (RobbieBuhite) [inline kzz7yh-f51825/cod-ve09v2_kzz7yh-f51825.xml:430]:
```suggestion
      Quaestio IIII. 
```

**Review by RobbieBuhite — CHANGES_REQUESTED** [review]

File: kzz7yh-f51825/cod-ve09v2_kzz7yh-f51825.xml (hunk 2 above)
Comment (evyryan) [reply kzz7yh-f51825/cod-ve09v2_kzz7yh-f51825.xml:95]: @jeffreycwitt

Comment (jeffreycwitt) [reply kzz7yh-f51825/cod-ve09v2_kzz7yh-f51825.xml:95]: sine is ok here
```suggestion
      <lb ed="#V" n="37"/>animalis. Sed accidens non est sine subiecto: ergo 
```

**Review by jeffreycwitt — APPROVED** [review]

**Pull request merged**